packages/jest-preset-default/scripts/setup-globals.js

```diff
@@ -5,8 +5,38 @@ global.window.tinyMCEPreInit = {
 	// <script> tag where it was loaded from, which of course fails here.
 	baseURL: 'about:blank',
 };
-global.window.requestAnimationFrame = setTimeout;
-global.window.cancelAnimationFrame = clearTimeout;
+
+global.window.requestAnimationFrame = function requestAnimationFrame(
+	callback
+) {
+	// eslint-disable-next-line no-restricted-syntax
+	const randomDelay = Math.round( ( Math.random() * 1_000 ) / 60 );
```

Comment:
Unrelated Aside: I'm wondering if the time period for animation frame becomes `1000 / 120` when the display has 120Hz like Apple devices with Pro Motion 😄

Reply:
this also shouldn't matter that much for the tests. by specification yes, we could get 120Hz > 8ms inter-frame times, but for the sake of mocking this function it will appear as if our fake display is 60Hz. code should not rely on the timing of the call but rather the `DOMHighResTimeStamp` passed into the callback if they want to perform time-based calculations.